doc/adr/0008-TimeRangeData-widget.md

```diff
@@ -0,0 +1,54 @@
+# 8. Time Range Data Widget
+
+Date: 2023-1-10
+
+## Status
+
+New
```

Comment:
Should the status be `Accepted` before merging it? However, the decision section needs to state the decision for that.

Reply:
Right. Above, `New` should have read `Proposed`, based [on this](https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions). Beside the possible `Accepted` (with documented decision indeed), one may leave the ADR as `Proposed`, for a decision to be finished later on. Could be mergeable saying so then, with a TBD decision.

Reply:
Status has been updated to "Accepted".